odl/space/pspace.py

```diff
@@ -281,6 +281,11 @@ def __len__(self):
         """
         return len(self.spaces)
 
+    @property
+    def nbytes(self):
+        """Total number of bytes used by an element of this space in memory."""
```

Comment:
Could be my formulation, but better would be "Total number of bytes in memory used by an element of this space."
The element property already does it like this.